app/jobs/content_depositor_change_event_job.rb

```diff
@@ -30,18 +27,19 @@ def log_work_event(work)
   end
   alias log_file_set_event log_work_event
 
-  def work
-    @work ||= Hyrax::ChangeContentDepositorService.call(repo_object, depositor, reset)
-  end
-
   # overriding default to log the event to the depositor instead of their profile
+  # and to log the event for both users
   def log_user_event(depositor)
-    # log the event to the proxy depositor's profile
-    proxy_depositor.log_profile_event(event)
+    previous_owner.log_profile_event(event)
     depositor.log_event(event)
   end
 
-  def proxy_depositor
-    @proxy_depositor ||= ::User.find_by_user_key(work.proxy_depositor)
+  private def previous_owner
```

Comment:
Yeah, I'm not sure when. I've been thinking it would be nice to use it because it's more explicit than a `private` floating around somewhere in the file, especially if extra indentation isn't enforced. But it's not idiomatic; I'll change it.